setup.py

```diff
@@ -55,7 +55,7 @@
                         "pythesint",
                         "urllib3",
                         "numpy",
-                        "gdal<3.0",
+                        "gdal",
```

Comment:
please pinpoint the version number with == in order not get caught in the famous 'The matrix of Hell'.

You or @aperrin66 may have other opinions based on the slim nansat-base!